content/health-supply-hub/submitting_bids.md

```diff
@@ -87,28 +87,29 @@ It is your sole responsibility to engage with the relevant Purchaser to indicate
+| F      | Number of Packs req uired in the Purchaser’s requested pack size                                    |
+| G      | Purchaser's requested Pack Size                                                                     |
+| H      | Total Quantity of single units being requested by the Purchaser                                    |
+| I      | Any conditions the Purchaser may have for a particular Quote Item                                   |
+| J      | Purchasers Unit of Measure                                                                          |
 
 1. This screen shot shows the fields you must fill in to submit a quote
 
 ![Quote Template Supplier Section](/health-supply-hub/images/quote_supplier_fields.png)
 
 | Column | Description                                                                                                   |
 | ------ | ------------------------------------------------------------------------------------------------------------- | 
-| J      | Enter in how many packs you are offering to supply the Purchaser                                              |
-| K      | Enter in the Pack Size of the product your offering to supply                                                 |
-| L      | Enter in the Price you are offering to supply a single pack of your product at, in the currency you allocated in the next column                                           |
-| M      | The currency your Quoted price per pack is in (Please use [ISO currency codes](https://en.wikipedia.org/wiki/ | ISO_4217#List_of_ISO_4217_currency_codes))                             |
-| N      | When your product is expected to expire, or the shelf life if the product has not yet been manufactured       |
-| O      | Delivery time, explain how long it would take to get your product to the Purchaser's identified delivery      | location                                                               |
-| P      | The finished product manufacturer of your product. NB Please check this is not just the regulatory sponsor / distributor of the product                                             |
-| Q      | How you would deliver the product eg Air freight or Sea Freight                                               |
-| R      | You can choose to put your internal reference code or product identifier here                                 |
-| S      | A field to put any further comments in to explain what you are offering the Purchaser more clearly, or include any other information you think the Purchaser may need to know |
+| K      | Enter in how many packs you are offering to supply the Purchaser                                              |
+| L      | Enter in the Pack Size of the product your offering to supply                                                 |
+| M      | Enter in the Price you are offering to supply a single pack of your product at, in the currency you allocated in the next column                                           |
```

Comment:
```suggestion
| M      | Enter in the Price you are offering to supply a single pack of your product at, in the currency you specify in the next column                                           |
```
Some of this wording a is a bit convoluted, thought this might be slightly tidier...